Src/System.Linq.Dynamic.Tests/EntitiesTests.cs

```diff
@@ -189,10 +189,10 @@ public void Entities_Where_ByDateTime_AsTypeAccess()
             var expected = _context.Posts.Where(x => x.PostDate > utcNow).ToArray();
 
             //Act
-            var test = _context.Posts.Where("PostDate > DateTime(\"" + utcNow + "\")").ToDynamicArray();
+            var test = _context.Posts.Where("PostDate > DateTime(\"" + utcNow.ToString("u") + "\")").ToDynamicArray();
 
             //Assert
-            CollectionAssert.AreEqual(expected, test);
+            CollectionAssert.AreEqual(expected, test, string.Format("expected: {0} found: {1}", expected.Length, test.Length));
```

Comment:
Actually these tests were never failed. I was tried to run the separately and in batch and still same result. I will continue to dig into.